introduced neo4j package for version 3.1.2

unterstein opened the pull request:
did the following:

copied neo4j/0 to neo4j/1 and replaced 3.1.0 with 3.1.2
copied neo4j-replica/0 to neo4j/1 and replaced 3.1.0 with 3.1.2

Fixes #991

joerg84 commented:
LGTM

joerg84 commented on repo/packages/N/neo4j-replica/1/resource.json:
```diff
@@ -0,0 +1,14 @@
+  "assets": {
+    "container": {
+      "docker": {
+        "dcos-neo4j": "neo4j/neo4j-dcos:1.0.0-3.1.1"
```
Does it make sense to have the same version schema for the docker image than for the package version above?

unterstein replied:
ah, in the docker image version `1.0.0` and `3.1.1` are swapped...

ryadav88 commented:
@unterstein does the example need to updated to reflect these changes? 
https://github.com/dcos/examples/tree/master/1.8/neo4j

unterstein commented:
Hi @ryadav88,

sorry for the delayed answer. No, no documentation change needed, just a bugfix release. Updated to newer 3.1.2 bugfix release in the meantime. Just replaced 3.1.1 with 3.1.2, hope this is not a problem.

Thanks
Johannes
